private/bufpkg/bufcheck/buflint/internal/buflintcheck/buflintcheck.go

```diff
@@ -275,14 +275,16 @@ func checkFieldLowerSnakeCase(add addFunc, field bufprotosource.Field) error {
 	name := field.Name()
 	expectedName := fieldToLowerSnakeCase(name)
 	if name != expectedName {
+		var otherLocs []bufprotosource.Location
+		if field.Extendee() == "" {
+			// also check the message for this comment ignore
+			// this allows users to set this "globally" for a message
+			otherLocs = []bufprotosource.Location{field.ParentMessage().Location()}
+		}
```

Comment:
This is really an open question: **Should** a message-level ignore comment apply to both regular fields _and_ extensions inside the message?

This branch currently does _not_ apply the comment to extensions at all. But I could easily change `if field.Extendee() == ""` to instead be `if field.ParentMessage() != nil`, and then the message-level ignores also apply to nested extensions.

Thoughts?

Reply:
I don't think message ignores should apply to extensions - they don't apply currently right?

Reply:
> they don't apply currently right?

Oh, that's a great point: they actually _do_ apply to extensions nested inside the message. So we should preserve that property for backwards compatibility.